File: code-grader-standalone/src/systest/data/inPlace/workArea/computeSqrt.cpp
#include <iostream>
#include <iomanip>
#include <cmath>

using namespace std;

int main()
{
    double x;
    cin >> x;
    double y = sqrt(x);
    cout << "The square root of " << setprecision(2) << x << " is "
         << (int)y << endl;
    return 0;
}

File: code-grader-standalone/src/systest/data/tutorials/simpleJavaParams/submissions/buggy/computeSqrt.java
public class computeSqrt {

    public static void main(String[] args) {
        Double x = Double.parseDouble(args[0]);
        int y = (int)Math.sqrt(x);
        System.out.printf("The square root of %.2f is %.4f%n", x, y);
    }

}
